suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -216,13 +218,17 @@ export const createDescriptorPreloadedDiscoveryThunk = createThunk(
         const networks = areTestnetsEnabled ? supportedNetworkSymbols : supportedMainnetSymbols;
 
         const availableCardanoDerivations = await dispatch(
-            getAvailableCardanoDerivationsThunk({ deviceState, device }),
+            getAvailableCardanoDerivationsThunk({
+                deviceState,
+                device,
+                isUseEmptyPassphraseForced: true,
+            }),
         ).unwrap();
 
         dispatch(
             createDiscovery({
                 deviceState,
-                authConfirm: !device.useEmptyPassphrase,
+                authConfirm: false,
```

Comment:
also comment here why authConfirm: false is used?

Reply:
authConfirm is parameter that controls the passphrase. Since these two changes are in `suite-native` where we do not want to use empty passphrase so far, I think it is selfexplenaory. Wha do you think?